newdle/client/src/components/creation/timeslots/Timeline.js

```diff
@@ -355,6 +355,32 @@ export default function Timeline({date, availability, defaultMinHour, defaultMax
     setHourSpan(getHourSpan(input));
   }, [candidates, defaultHourSpan, defaultMaxHour, defaultMinHour, duration]);
 
+  const creationTimezone = localStorage.getItem('creationTimezone');
+  const revertIcon = (
+    <Popup
+      trigger={
+        <Icon
+          name="redo"
+          color="grey"
```

Comment:
```suggestion
          color="white"
```

The contrast with grey is pretty bad :/